### PR TITLE
Added Hacktoberfest Banner to home page

### PR DESCRIPTION
**Pull request by Burnsy**
image

Built a quick banner for Hacktoberfest, I could not get it dev working locally so it's built blind and in-browser!  

But it should look the same!

### CODE REVIEW

**Comment by thedavidprice** [comment]
Just a quick look for now. Will review soon. Thank you @Burnsy 🚀

What do you think about linking to this Issue?

https://github.com/redwoodjs/redwood/issues/1266

**Comment by Burnsy** [comment]
It already is linked to it!

**Comment by thedavidprice** [comment]
@Burnsy @cannikin What do you think about this:

Screen Shot 2020-10-04 at 9 07 58 AM
---
Screen Shot 2020-10-04 at 9 11 08 AM

### Here's what I did:
- increased padding-top/bottom from 3 to 5
- font extra-bold
- replaced button text (note: "RedwoodJS", no `.`)
- on mobile, centered top text
- Change link to this Issue in Framework: https://github.com/redwoodjs/redwood/issues/1266

**Comment by Burnsy** [comment]
Looks great! 🔥

**Review by cannikin — APPROVED** [review]

**Comment by thedavidprice** [comment]
@Burnsy To be clear, I **did not** make any code changes. That's just a mockup from the browser.

Given that everyone's agreed on the display, can you please make the changes in the code? There's no way I can get to to today. Thanks in advance!

File: code/html/layouts/application.html [hunk 1]
```diff
@@ -26,6 +26,31 @@
 
   <body class="relative bg-white text-gray-900 font-sans antialiased h-full flex flex-col" data-controller="application" data-action="turbolinks:before-cache@document->application#saveScrollPosition turbolinks:before-cache@document->application#closeNav keydown@window->application#focusSearch">
     <div class="flex-grow flex-shrink-0 flex-auto">
+      <div class="bg-hacktoberfest-blue-500">
```
Comment (thedavidprice) [inline code/html/layouts/application.html:29]: Make styling changes per:
https://github.com/redwoodjs/redwoodjs.com/pull/394#issuecomment-703278572

**Review by thedavidprice — CHANGES_REQUESTED** [review]
See inline comment re:
https://github.com/redwoodjs/redwoodjs.com/pull/394#issuecomment-703278572

**Comment by Burnsy** [comment]
Done 🎉  @thedavidprice

**Comment by thedavidprice** [comment]
ship it squirrel

**Pull request merged**